packages/core/src/browser/preferences/preference-service.ts

```diff
@@ -216,7 +216,7 @@ export class PreferenceServiceImpl implements PreferenceService {
         }
     }
 
-    protected getProvider(scope: PreferenceScope): PreferenceProvider | undefined {
+    getProvider(scope: PreferenceScope): PreferenceProvider | undefined {
```

Comment:
i could not find any external client for it, revert to keep it protected?

Reply:
Yes, I used that at one time in TaskConfigurationManager in order to access user-scope preferences, but since changed it to use @named(PreferenceScope.User). I'll revert the change.